octobot/tests/messenger_test.rs

```diff
@@ -137,6 +162,7 @@ fn test_sends_to_assignees() {
         &[github::User::new("assign1"), github::User::new("assign2")],
         "",
         &Vec::<github::Commit>::new(),
+        vec!["http://the-github-host/some-user/some-repo/pulls/1".to_string()],
```

Comment:
we aren't using urls for the guid anymore, are we? i think we should update the tests for clarity.